docs/using-the-jdbc-driver/using-plugins/UsingTheFederatedAuthPlugin.md

```diff
@@ -0,0 +1,54 @@
+|-------------------------|:------:|:--------:|:--------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------|--------------------------------------------------------|
+| `idpHost`               | String |   Yes    | The hosting URL of the Identity Provider.                                                                                                                     | `null`                   | `ec2amaz-ab3cdef.example.com`                          |
+| `idpPort`               | String |    No    | The hosting port of Identity Provider.                                                                                                                        | `443`                    | `1234`                                                 |
+| `rpIdentifier`          | String |    No    | The relaying party identifier.                                                                                                                                | `urn:amazon:webservices` |                                                        |
+| `iamRoleArn`            | String |   Yes    | The ARN of the IAM Role that is to be assumed.                                                                                                                | `null`                   | `arn:aws:iam::123456789012:role/adfs_example_iam_role` |
+| `iamIdpArn`             | String |   Yes    | The ARN of the Identity Provider.                                                                                                                             | `null`                   | `arn:aws:iam::123456789012:saml-provider/adfs_example` |
+| `iamRegion`             | String |   Yes    | The IAM region where the IAM token is generated.                                                                                                              | `null`                   | `us-east-2`                                            |
+| `federatedUserName`     | String |   Yes    | The federated user name.                                                                                                                                      | `null`                   | `jimbob@example.com`                                   |
+| `federatedUserPassword` | String |   Yes    | The federated user password.                                                                                                                                  | `null`                   | `someRandomPassword`                                   |
+| `user`                  | String |   Yes    | The user name of the IAM user with access to your database. Note: If you have previously used the IAM Authentication Plugin, this would be the same IAM user. | `null`                   | `some_user_name`                                       |
```

Comment:
Can we link to AWS docs here? https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/UsingWithRDS.IAMDBAuth.Connecting.html

Reply:
Linked!